local/bin/1-prepare-space

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+# stop will check the dir, make sure PCFDev is running and the `cf` binary is installed
+./bin/stop
+
+docker-compose up -d pg conjur
```

Comment:
I would actually split this into a separate "deploy-conjur" step or something just to emphasize that it's something that needs to be done. Gives the SE folks a good place to talk about it when running through the demo I would think.

Reply:
I renamed it to `1-start-conjur-and-prep-space` - I don't want to get too granular with these scripts, if possible